peekingduck/configloader.py

```diff
@@ -43,6 +46,46 @@ def _get_config_path(self, node: str) -> Path:
 
         return file_path
 
+    @staticmethod
+    def _load_mapping(node_name: str) -> Dict[str, int]:
+        """Loads class name to object ID mapping from the file
+        peekingduck/pipeline/nodes/model/master_map.yml
+
+        Args:
+            node_name (str): Tells function which mapping to load,
+                             Possible values = { model.efficientdet, model.yolo }.
+
+        Returns:
+            Dict[str, int]: Mapping of class names to object IDs relevant to given node_name
+
+        Tech Notes:
+            master_map.yml comprises two documents:
+            1. model mapping: tells which mapping system a particular object detection model uses
+            2. class name to ID mapping: maps class name to ID, supports multiple mapping systems
+        """
+        assert node_name in (
+            "model.yolo",
+            "model.efficientdet",
+        ), f"Name Error: expect model.yolo or model.efficientdet but got {node_name}"
```

Comment:
Currently this only allows `yolo` and `efficientdet` but the master mapping yaml suggests that it works with `yolox` as well. Would it be better to check against the keys in `model_map_dict` instead of hard coding it here?

Reply:
Good idea: generated `node_name_list` from `model_map_dict` and checked against that instead of hardcoding. Easier on maintenance too.

Reply:
Haven't tested with `model.yolox` yet; am assuming it will work since it uses same numbering system as `yolo`.